secure/storage/src/vault.rs

```diff
@@ -1,7 +1,8 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::{Capability, Error, Identity, Policy, Storage, Value};
+use crate::kv_storage::KVStorage;
+use crate::{Capability, Error, Identity, Policy, Value};
```

Comment:
where is your merge? :P

Reply:
It died.. along with my hope of these things being detected automatically.. :(

Done, thanks!